spec/translation_quality/dictionaries.yaml

```diff
@@ -232,6 +232,59 @@ x-paths:
           source: |
             curl -X DELETE -H "Authorization: Bearer $smartlingToken" -H "Content-Type: application/json" https://api.smartling.com/translation-quality-api/v2/users/current/settings/dictionaries/$smartlingLocaleId/$wordUid
 
+
+  dictionary_regex_check_rule_types:
+    get:
+      description: >-
+        Get custom quality check rule types that are available.
+      summary: Get custom quality check rule types
+      tags:
+        - Translation quality checks
+      operationId: getRegexQualityCheckRuleTypes
+      produces:
+        - application/json
+
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DictionaryRegexCheckRuleTypesResponse'
```

Comment:
why do we use both camel case and snake case?

Reply:
$ref: '#/components/schemas/DictionaryRegexCheckRuleTypesResponse'
$ref: '#/components/examples/regex_check_rule_types_example'